CONTRIBUTING.md

```diff
@@ -162,20 +150,36 @@ cogito/
 └── test_repo_owner
 ```
 
-## Running the end-to-end tests
+## Running the integration tests
 
-We are finally ready to run also the end-to-end tests:
+We are finally ready to run also the integration tests:
 
-```console
-$ task test
+```
+$ task test-e2e
 ```
 
-The end-to-end tests have the following logic:
+The integration tests have the following logic:
 
 * If none of the environment variables are set, we skip the test.
 * If all of the environment variables are set, we run the test.
 * If some of the environment variables are set and some not, we fail the test. We do this on purpose to signal to the user that the environment variables are misconfigured.
 
+## Running a specific end-to-end test
+
+Use the `test:env` task target, that runs a shell with available all the secrets needed for the e2e tests.
```

Comment:
Here they are referred to as "end-to-end"...

I don't really care if we call them integration or end-to-end. Of course, `e2e` is a tempting name as it's much shorter. I would just like that we settle on a single naming.

Reply:
You are right. Historically, I was calling them e2e, but that name is somehow misleading, so I renamed to integration, but I missed some. Will fix.

Reply:
done

Reply:
done